src/openxr/extensions/xr_fb_display_refresh_rate_extension_wrapper.cpp

```diff
@@ -0,0 +1,28 @@
+#include "xr_fb_display_refresh_rate_extension_wrapper.h"
+
+XRFbDisplayRefreshRateExtensionWrapper *XRFbDisplayRefreshRateExtensionWrapper::singleton = nullptr;
+
+XRFbDisplayRefreshRateExtensionWrapper *XRFbDisplayRefreshRateExtensionWrapper::get_singleton() {
+    if (!singleton) {
+        singleton = new XRFbDisplayRefreshRateExtensionWrapper();
+    }
+
+    return singleton;
+}
+
+XRFbDisplayRefreshRateExtensionWrapper::XRFbDisplayRefreshRateExtensionWrapper() {
+    openxr_api = OpenXRApi::openxr_get_api();
+}
+
+XRFbDisplayRefreshRateExtensionWrapper::~XRFbDisplayRefreshRateExtensionWrapper() {
+    cleanup();
+    OpenXRApi::openxr_release_api();
+}
+
+std::map<const char *, bool *> XRFbDisplayRefreshRateExtensionWrapper::get_request_extensions() {
+    return request_extensions;
+}
+
+void XRFbDisplayRefreshRateExtensionWrapper::cleanup() {
+
+}
```

Comment:
Was the rest of the implementation missing here for a specific reason or did you not get around to it? Happy to do this once merged.